app/assets/javascripts/listings/templates/listing/_pre_lottery_info.html.slim

```diff
@@ -6,7 +6,7 @@
       | {{ $ctrl.parent.listing.Lottery_Date | date : 'longDate' }}
     span.content-group_time
       | {{ $ctrl.parent.listing.Lottery_Date | date : 'shortTime' | lowercase }}
-  p.content-group_address.c-steel.skiptranslate
+  p.content-group_address.c-steel.skiptranslate ng-if="!$ctrl.parent.lotteryVenueAvailable()"
```

Comment:
This logic doesn't look quite right. Seems like we want to show if we show covid update is false and lottery venue info is available. 
```suggestion
  p.content-group_address.c-steel.skiptranslate ng-if="!$ctrl.parent.showCovidUpdate && $ctrl.parent.lotteryVenueAvailable()"
```